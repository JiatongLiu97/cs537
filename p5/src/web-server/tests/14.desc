Mix of static and  dynamic requests (checking stat_process)
