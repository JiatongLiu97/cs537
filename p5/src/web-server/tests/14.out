Total requests: 40; Total static: 20; Total dynamic: 20
